docs: document ES|QL Fast mode toggle for approximation (#7141) (#7143)
elastic/docs-content: explore-analyze/dashboards/using.md

This section shows the most common ways for you to filter dashboard data. For more information about {{kib}} and {{es}} filtering capabilities, refer to [](/explore-analyze/query-filter.md).

:::{note}
:applies_to: {"stack": "preview 9.5", "serverless": "preview"}

> When a dashboard includes {{esql}} visualizations that use the `STATS` command, you can turn on {icon}`bolt` **Fast mode** to return faster, estimated results for these visualizations. Refer to [](/explore-analyze/query-filter/languages/esql-kibana.md#approximation-fast-mode).

Reviewer: Would it make sense to stats info, or is that obvious at this point?

Author: I think it's fine, the detail, including the relation with STATS, is on the linked page

> When a dashboard includes an {{esql}} visualization that uses `STATS`, you can turn on Fast mode from the {icon}`bolt` button next to the query bar to return faster, estimated results on large datasets. Refer to [Get faster results with Fast mode](/explore-analyze/query-filter/languages/esql-kibana.md#approximation-fast-mode).

Author: ```suggestion
When a dashboard includes {{esql}} visualizations that use the `STATS` command, you can turn on {icon}`bolt` **Fast mode** to return faster, estimated results for these visualizations. Refer to [Get faster results with Fast mode](/explore-analyze/query-filter/languages/esql-kibana.md#approximation-fast-mode).
```
